### DISCUSSION TITLE
ComfyUI support

### DISCUSSION THREAD

**Discussion by yoni333** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#158] [Q&A] [answered]
Can it help also for ComfyUI? 
Is there is a guide for that?
Nativly comfyui is much more faster than automatic111

**Comment by phineas-pta** [comment, marked as answer]
#5

**Comment by yoni333** [comment]
node for this driver
https://github.com/comfyanonymous/ComfyUI_TensorRT
